.env.example

```diff
@@ -68,6 +68,8 @@ PREVENT_ACCESSING_MISSING_ATTRIBUTES=false
 PRINT_COST_ONESIDED=8
 PRINT_COST_TWOSIDED=12
 PRINTER_NAME=NemUjBela
+PRINTER_ADDITIONAL_ARGS=
+PRINTER_STAT_ADDITIONAL_ARGS=
```

Comment:
I'm not sure how much of the changes to printing will be necessary once the printer server is redesigned (eotvoscollegium/smaugreinstall#8). I'm planning on tackling that task on Thursday; could we delay this part until then?